src/main/java/digital/slovensko/autogram/core/CustomSynchronizationStrategy.java

```diff
@@ -0,0 +1,21 @@
+package digital.slovensko.autogram.core;
+
+import eu.europa.esig.dss.model.tsl.TLInfo;
+import eu.europa.esig.dss.tsl.sync.ExpirationAndSignatureCheckStrategy;
+
+import java.util.Date;
+
+public class CustomSynchronizationStrategy extends ExpirationAndSignatureCheckStrategy {
+	@Override
+	public boolean canBeSynchronized(TLInfo trustedList) {
+        var signingCertificate = trustedList.getValidationCacheInfo().getSigningCertificate();
+
+        if (signingCertificate == null)
```

Comment:
pre ten Belgický TL je tu signingCertificate null, takže zjavne už pri parsovaní LOTL sa zistí, že ten cert je exspirovaný a ignoruje sa. Ale toto je bola všeobecne najlepšia cesta - takáto CustomStrategy - písali aj v DSS Jire